src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -167,15 +167,21 @@
   <data name="Unauthorized" xml:space="preserve">
     <value>Authentication failed.</value>
   </data>
-  <data name="UnsupportedAcceptHeader" xml:space="preserve">
-    <value>The requested "accept" header is not supported.</value>
-  </data>
-  <data name="UnsupportedContentTypeHeader" xml:space="preserve">
-    <value>The requested "content-type" header is not supported.</value>
-  </data>
   <data name="UnsupportedFormatParameter" xml:space="preserve">
     <value>The requested "_format" parameter is not supported.</value>
   </data>
+  <data name="UnsupportedHeader" xml:space="preserve">
+    <value>The requested "{0}" header is not supported.</value>
+    <comment>{0} is the header name.</comment>
+  </data>
+  <data name="UnsupportedOperation" xml:space="preserve">
+    <value>The requested {0} operation is not supported.</value>
```

Comment:
Might want to put quotes around the placeholder like we're doing with `UnsupportedHeader`.